### PR TITLE
javascript-custom-methods - Marianne Cuevas

### PR DESCRIPTION
**Pull request by mariannecuevas**
image

### CODE REVIEW

File: javascript-custom-methods/calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    for (var i = 0; i < numbers.length; i++) {
+      sum += numbers[i];
+    }
+    return sum;
+  },
+  getAverage: function (numbers) {
+    var sum = 0;
+    for (var i = 0; i < numbers.length; i++) {
+      sum += numbers[i];
+      var average = sum / numbers.length;
```
Comment (scrumptiousdev) [inline javascript-custom-methods/calculator.js]: Small suggestion — with this, average is being defined every time loop loops. What we can do is just return `sum / numbers.length` outside of loop when `sum` is all done calculating

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Very small change request

File: javascript-custom-methods/calculator.js (hunk 1 above)
Comment (mariannecuevas) [reply javascript-custom-methods/calculator.js]: Got it!

**Review by scrumptiousdev — APPROVED** [review]
Approved! :)

**Pull request merged**